src/pages/Vaults/Vault/RequestIssueModal/index.tsx

```diff
@@ -239,6 +245,10 @@ const RequestIssueModal = ({ onClose, open, collateralIdLiteral, vaultAddress }:
       return 'Invalid BTC amount input!';
     }
 
+    if (btcToGovernanceTokenRate.toBig().eq(0)) {
```

Comment:
hi @peterslany PR looks good but I have a query about this - is there a way to make it clear why checking `btcToGovernanceTokenRate.toBig().eq(0)` means that the oracle is offline? Or a way to use this to set a variable in the app somewhere so that we can do something like `const oracleOffline = btcToGovernanceTokenRate.toBig().eq(0)`

No need to spend a lot of time on this, the PR's good as it is (and consistent with what we're doing elsewhere) but if there's anything quick then do that :)

Reply:
Good point, I'll move it into separate variable and add a comment for the clarity : )